### DISCUSSION TITLE
Mix 2 sound banks

### DISCUSSION THREAD

**Discussion by Sam11100Narb** [GrandOrgue/grandorgue#1542] [Developers]
Hello, 

For a customer, I'm looking for a technical solution :
On the same midi console, my customer want an organ and a clavichord (to create a midi claviorganum).
Is it possible actually ? How ?
Many thanks.

Samuel Poujade
Les Claviers du Vent

**Comment by larspalo** [comment]
Normally, the Claviorganum is the combination of a Harpsichord and an Organ. There is one available by Pere Casulleras on the [Sample Sets wiki page](https://github.com/GrandOrgue/grandorgue/wiki/Sample-Sets).

If you want to create your own combination of an organ and a harpsichord (or other keyboard instrument) from other already existing sample sets you'll have to combine them yourself in the custom .organ file. For such a task, I think you could find [GOODF](https://github.com/larspalo/GOODF) helpful.
